Gems/Atom/Feature/Common/Code/Source/PostProcessing/TaaPass.cpp

```diff
@@ -69,10 +69,29 @@ namespace AZ::Render
 
         m_shaderResourceGroup->SetConstant(m_constantDataIndex, cb);
 
-
         Base::CompileResources(context);
     }
-    
+
+    void TaaPass::BuildCommandListInternal(const RHI::FrameGraphExecuteContext& context)
+    {
+        Base::BuildCommandListInternal(context);
+        if (ShouldCopyHistoryBuffer)
+        {
+            context.GetCommandList()->Submit(m_copyItem);
+        }
+    }
+
+    void TaaPass::SetupFrameGraphDependencies(RHI::FrameGraphInterface frameGraph)
+    {
+        if (ShouldCopyHistoryBuffer)
+        {
+            // Override the estimated item count to include the copy item.
+            frameGraph.SetEstimatedItemCount(2);
+        }
+
+        Base::SetupFrameGraphDependencies(frameGraph);
```

Comment:
I think you need to change the order of these two calls. 
The Base::SetupFrameGraphDependencies will call frameGraph.SetEstimatedItemCount(1); which override your frameGraph.SetEstimatedItemCount(2);

Reply:
While technically not required since `SetEstimatedItemCount()` isn't called by any of the base classes, I'll go ahead and change this just in case they are updated some day to do that.